tests/e2e/backup_restore_suite_test.go

```diff
@@ -1,4 +1,4 @@
-package e2e
+package e2e_test
```

Comment:
qq: Is the name change intentional or accidental ? The PR description does not mention about `e2e_test` package https://github.com/openshift/oadp-operator/pull/531#issue-1100332242

Reply:
@shubham-pampattiwar yes that is intentional change. Just wanted to be more specific here, to reflect that its a test package (taken as reference from Ginkgo docs)